README.md

```diff
@@ -1,3 +1,29 @@
+````
+-----------------------------          _______  _______  _______  _______ ------------------------------
+-----------------------------|\     /|(  ____ \(  ___  )(  ____ \(  ____ \------------------------------
+-----------------------------| )   ( || (    \/| (   ) || (    \/| (    \/------------------------------
+-----------------------------| |   | || (__    | |   | || (_____ | (_____ ------------------------------
+-----------------------------( (   ) )|  __)   | |   | |(_____  )(_____  )------------------------------
+----------------------------- \ \_/ / | (      | |   | |      ) |      ) |------------------------------
+-----------------------------  \   /  | )      | (___) |/\____) |/\____) |------------------------------
+-----------------------------   \_/   |/       (_______)\_______)\_______)------------------------------
+                                   Vanderbilt Free & Open Source Society
+````                                             
+
+<p align="center"><strong>Welcome to Vandy's Open Source One Stop Shop</strong></p>
+
+## Getting Started
+
+If you're new, read our [Guide for New Members](NEW_MEMBER_GUIDE.md).
+
+If you're not, continue below to our Bounty List.
```

Comment:
Think I'd just call it our projects. It's not a bounty list because there isn't any payout

Reply:
changed